src/components/SQForm/SQFormRadioButtonGroupItem.tsx

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import Radio from '@material-ui/core/Radio';
-import {makeStyles} from '@material-ui/core/styles';
-import FormControlLabel from '@material-ui/core/FormControlLabel';
+import {makeStyles, Theme} from '@material-ui/core/styles';
+import {FormControlLabel, RadioProps} from '@material-ui/core';
 
-const useStyles = makeStyles(theme => ({
+interface SQFormRadioButtonGroupItemProps {
+  /** Value of the radio button */
+  value: string | boolean | number;
+  /** Label for the radio button */
+  label: string;
+  /** Whether this radio button is disabled */
+  isDisabled?: boolean;
+  /** Whether the group this button is in is displayed in a row */
+  isRowDisplay?: boolean;
+  /** Props for the radio input */
+  inputProps?: RadioProps;
+}
+
+const useStyles = makeStyles((theme: Theme) => ({
```

Comment:
The type for the `theme` argument is automatically provided by `makeStyles`, so there's no need to give it the `Theme` type again.
![image](https://user-images.githubusercontent.com/35306025/125716220-6a9147ce-c9ac-4faf-8d23-8e0eee24072b.png)

Reply:
Great catch Cody, I went ahead and updated this!